iib/workers/tasks/opm_operations.py

```diff
@@ -515,13 +515,20 @@ def deprecate_bundles_fbc(
     :param str binary_image: binary image to be used by the new index image.
     :param str from_index: index image, from which the bundles will be deprecated.
     """
+    conf = get_worker_config()
     index_db_file = _get_or_create_temp_index_db_file(base_dir=base_dir, from_index=from_index)
 
-    opm_registry_deprecatetruncate(
-        base_dir=base_dir,
-        index_db=index_db_file,
-        bundles=bundles,
-    )
+    # Break the bundles into chunks of at max iib_deprecate_bundles_limit bundles
+    for i in range(
+        0,
+        len(bundles),
+        conf.iib_deprecate_bundles_limit,
+    ):  # Determine position i in the bundles array
+        opm_registry_deprecatetruncate(
```

Comment:
Example: we split input to 4 chunks. What happens when last chunk fails? 
Will the `IIBError` from `run_cmd` terminate everything? 
I am asking this to verify that we do not end up in inconsistent deprecation (3 out of 4 chunks were deprecated and image was build with it)

Reply:
I suppose it would work as previously - the image would not be built if there would be a failure during `opm_registry_deprecate_truncate`.